### PR TITLE
add week-8 and initial readme to master-reference

### PR DESCRIPTION
**Pull request by Akin909**
add initial readme to master-reference with a skeleton week schedule which we will update with the weeks content as we develop them. 
relates #463

### CODE REVIEW

**Comment by jsms90** [comment]
@PiotrBerebecki @Akin909 great to see you guys making a start :tada: 

Please have a read through the [contributing guidelines](https://github.com/foundersandcoders/master-reference/blob/master/CONTRIBUTING.md) of this repo, particularly with respect to:
+ who to assign to PRs
+ how to submit a [pull request](https://github.com/foundersandcoders/master-reference/blob/master/CONTRIBUTING.md#pull-requests) i.e. what goes in the body

**Comment by Akin909** [comment]
@jsms90, I've updated the body of the PR according to the contributing.md please let me know if this is sufficient.

**Comment by jsms90** [comment]
Beautiful! Thanks @Akin909 :tada:

File: coursebook/week-8/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# EXPRESS AND HANDLEBARS WEEK
+
+Schedule
+==
+
+### Day 1:
+
+10:00-10.15 - Introduction to gitbook, learning outcomes & project teams.
```
Comment (jsms90) [inline coursebook/week-8/README.md]: Project teams will already have been chosen in a prior week, so this isn't relevant to week 8 :+1:

File: coursebook/week-8/README.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+Start each week with a mission and a goal.
```
Comment (jsms90) [inline coursebook/week-8/README.md]: This is just a note in the general week schedule for mentors really. Prob isn't needed in each individual week's readme

**Pull request merged**